app/src/main/res/values-sw360dp-v13/preference.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why is this in `values-sw-360dp-v13`? Couldn't it just be in `values`?

Reply:
I tried it putting it in values but it didn't work. You can refer to this thread, https://stackoverflow.com/questions/18509369/android-how-to-remove-margin-padding-in-preference-screen